src/organisms/subject-attend.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from 'react';
+import AttendanceStatus from '../molecules/attendance-status';
+import { API_ROOT } from "@/queries/config";
+
+type AttendancesSum = {
+  onTime: number;
+  late: number;
+  miss: number;
+};
+
+const SubjectAttend = () => {
+    const queryParams = new URLSearchParams(window.location.search);
```

Comment:
```suggestion
import { API_ROOT } from "@/queries/config";
import { useSearchParams } from "next/navigation";

type AttendancesSum = {
  onTime: number;
  late: number;
  miss: number;
};

const SubjectAttend = () => {
    const queryParams = useSearchParams();
```
クエリパラメータの取得には `next/navigation` の `useSearchParams` フックを使ってください.